contributing.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to FingerprintJS Pro Google Tag Manager
```

Comment:
In the readme we talk about 2 products in this repository:
- FingerprintJS Pro Google Tag Manager template
- FingerprintJS Pro adapter for Google Tag Manager

Let's separate contribution rules for it. [Here](https://developers.google.com/tag-platform/tag-manager/templates/gallery#update_your_template) is the main information about updating GTM templates. (It automatically, but we need to fill the versions section in `metadata.yaml`). Also, it will be good to say that new features for the template are better to follow with tests, that can be run inside the template environment in GTM.

Reply:
Good idea, I've added a new section for the tag manager template. 

Do you think that we've provided enough information and that it's okay to link to GTM docs for more details, or should we add something more ourselves as well?